doc/model/train-fitting-population.md

```
@@ -0,0 +1,112 @@
+
+See the [preprint on arXiv](https://arxiv.org/abs/2606.01763) for details and an example of how the DeepPopulation model was used to study small polaron transport.
+
+In this example, we will show you how to train a model to fit the atomic charge population for a titanium dioxide system. A complete training input script of the examples can be found in
+
+```bash
+$deepmd_source_dir/examples/population/train/input_torch.json
+```
+
+The training data is composed of the first 50 frames of the published [LEOPOLD](https://zenodo.org/records/15302762) dataset of Birschitzky et al. used in the publication [Machine Learning Small Polaron Dynamics](https://doi.org/10.1103/PhysRevLett.134.216301). This data is licensed under the Apache License 2.0.
+
+Similar to the `input.json` used in `ener` mode, training JSON is also divided into {ref}`model <model>`, {ref}`learning_rate <learning_rate>`, {ref}`loss <loss>` and {ref}`training <training>`. Most keywords remain the same as `ener` mode, and their meaning can be found [here](dpa3.md). To fit the `dos`, one needs to modify {ref}`model[standard]/fitting_net <model[standard]/fitting_net>` and {ref}`loss <loss>`.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)